frontend/src/styles/_colors.scss

```diff
@@ -2,3 +2,4 @@ $background-color: #fefefe;
 $text-color: #100c08;
 $item-color: #ffffff;
```

Comment:
Change name of variables that specify background color
e.g $item-bg-color: #fff;